name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/AbstractPDFExtractor.java

```diff
@@ -105,8 +106,15 @@ public String getPDFAuthor()
             }
 
             for (Item item : items)
-                item.getSubject().setNote(filename);
-
+            {   
+                // Don't overwrite note set by the PDF extractor
+                Annotated subject = item.getSubject();
+                String note = subject.getNote();
+                if (note == null || note.isEmpty())
+                {
+                    subject.setNote(filename);
+                }
```

Comment:
Open for suggestions

Reply:
@Mauin - usually the note should be null after extraction, but in cases like yours, the filename should be appended to be consistent and the note to show, which document the import came from.

see PR https://github.com/buchen/portfolio/pull/1676